rmgweb/main/templates/resources.html

```diff
@@ -42,6 +42,11 @@ <h2>Publications:</h2>
         </li>
     </ol>
 
+    <h2>RMG Workshop Recordings:</h2>
+    <ol>
+        <li><a href="https://tinyurl.com/RMG2019">2019 January</a></li>
```

Comment:
You could consider putting the full playlist link here, in case tinyurl ever stops working. It is unlikely though, so you don't have to change this.

https://www.youtube.com/playlist?list=PLZUqt5RldKbTcbPdmDIvY5mfaWAerMUo7